packages/deployer/ens.js

```diff
@@ -151,7 +151,9 @@ class ENS {
 
     // Set top-level name
     let builtName = nameLabels[0];
-    await this.devRegistry.setSubnodeOwner("0x0", sha3(builtName), from, {
+    const ZERO_NODE =
+      "0x0000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
Should remove if "0x0" suffices (@spacesailor24)

Reply:
Should be doubled backed with latest base branch.

Reply:
"0x0" fails the validation (`Web3 validator found 1 error[s]:
value "0x0" at "/0" must pass "bytes32" validation`)